test/masterzTemplate.ts

```diff
@@ -166,6 +235,58 @@ describe('MasterZTemplate', function () {
         'Checkpoint not ready to start',
       )
     })
+
+    // test with non-admin account
+    itIf(integrationTestsOn)('Should start second checkpoint', async function () {
+      await deployProject()
+      await approveDonationToContract()
+      await (await masterzTemplateContract.connect(owner).donate(0, BigNumber.from('10'))).wait()
+      await (await masterzTemplateContract.connect(owner).startProject(0)).wait()
+      await (await masterzTemplateContract.connect(owner).startCheckPoint(0)).wait()
+      await (await masterzTemplateContract.connect(owner).finishCheckpoint(0)).wait()
+      await (await masterzTemplateContract.connect(owner).startCheckPoint(0)).wait()
+      const projects = await masterzTemplateContract.connect(owner).listProjects()
+      const checkpoints = projects[0].checkpoints
+      expect(projects[0].projectState).equals(2)
+      expect(projects[0].activeCheckpoint).equals(1) // check active checkpoint
+      expect(checkpoints[1].state).equals(2) // check checkpoint status
+    })
+
+    itIf(integrationTestsOn)('Should finish second checkpoint', async function () {
+      await deployProject()
+      await approveDonationToContract()
+      await (await masterzTemplateContract.connect(owner).donate(0, BigNumber.from('10'))).wait()
+      await (await masterzTemplateContract.connect(owner).startProject(0)).wait()
+      await (await masterzTemplateContract.connect(owner).startCheckPoint(0)).wait()
+      await (await masterzTemplateContract.connect(owner).finishCheckpoint(0)).wait()
+      await (await masterzTemplateContract.connect(owner).startCheckPoint(0)).wait()
+      await (await masterzTemplateContract.connect(owner).finishCheckpoint(0)).wait()
+      const projects = await masterzTemplateContract.connect(owner).listProjects()
+      const checkpoints = projects[0].checkpoints
+      expect(projects[0].projectState).equals(2)
```

Comment:
let's create a fixtures.ts file and create some enums for these values if the types are not generated